### PR TITLE
Things I did for a release.

### PR DESCRIPTION
**Pull request by pelson**
There is no scientific, rigorous approach here. This is what I did to get the pypi upload to work.
(plus a minor year update).

### CODE REVIEW

**Comment by QuLogic** [comment]
Makes sense to me. :+1:

File: setup.py [hunk 1]
```diff
@@ -347,10 +344,10 @@ def get_config_var(name):
                                                      remove='lib/cartopy/')) +\
                              list(file_walk_relative('lib/cartopy/data/shapefiles/gshhs',
                                                      remove='lib/cartopy/')) +\
-                             ['io/srtm.json']
+                             ['io/srtm.json'],
+                  '': ['requirements/*.txt'],
```
Comment (QuLogic) [inline setup.py]: I'm not entirely sure this one has any effect, though.

Comment (pelson) [inline setup.py]: The requirements weren't being included as part of the distribution, and therefore couldn't be used by PIP when installing.

Comment (QuLogic) [inline setup.py]: Hmm, yes, but was that not fixed by the `MANIFEST.in`? Or is this necessary too?

Comment (pelson) [inline setup.py]: It seemed not - but I was trialling and erroring quite a bit.

**Comment by pelson** [comment]
Looks like this has already been incorporated. I'll double check with the release.

**Pull request closed without merging**